src/api.js

```diff
@@ -1,3 +1,9 @@
+import axios from "axios";
+
 const itemsAPI = axios.create({
     baseURL: "https://news-api-project-hnma.onrender.com/"
-})
\ No newline at end of file
+})
+
+export function getAllArticles(){
+    return itemsAPI.get('/api/articles')
```

Comment:
It would be good to take the articles off the key off `data` and then off the key of `articles` in this function rather than your component, to keep your component code a bit cleaner